.github/workflows/snyk.yaml

```diff
@@ -0,0 +1,36 @@
+name: Snyk test Docker image
+on: 
+  push:
+    branches: [master]
+  workflow_dispatch:
+
+jobs:
+  snyk_test:
+    name: Snyk test
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
```

Comment:
Is there a reason to use `v3` over `v4`? I would suggest to use the newest supported version for new workflows.